architecture/core_concepts/routes.adoc

```diff
@@ -760,3 +760,31 @@ at a project/namespace level.
 When namespace labels are used, the service account for the router
 must have `cluster-reader` permission to permit the
 router to access the labels in the namespace.
+
+[[route-timeouts]]
+== Route specific timeouts 
+
+By default a server side timeout is set across all routes, it is possible to change that 
+timeout using an annotation in the route configuration.
+
+
+.A Route Setting Custom Timeout
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    router.openshift.io/haproxy.timeout: 5500ms <1>
```

Comment:
Should not the key name of the annotation be `haproxy.router.openshift.io/timeout`?

According to https://github.com/openshift/origin/blob/master/images/router/haproxy/conf/haproxy-config.template#L229